tests/facility/patient/encounter.spec.ts

```diff
@@ -0,0 +1,140 @@
+import { expect, test } from "@playwright/test";
+
+test.use({ storageState: "tests/.auth/user.json" });
+
+test.describe("Create an Encounter", () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto("/");
+    await page
+      .getByRole("link", { name: /facility with patients/i })
+      .first()
+      .click();
+  });
+
+  test("through patient home", async ({ page }) => {
+    await test.step("open the encounter", async () => {
+      await page.getByRole("button", { name: "Toggle Sidebar" }).click();
+      await page.getByRole("button", { name: "Patients", exact: true }).click();
+      await page.getByRole("link", { name: "All Encounters" }).click();
+
+      await page.getByText("Date").click();
+      await page.getByRole("button", { name: "Last 6 months" }).click();
+
+      await page.keyboard.press("Escape");
+
+      // Wait for the first patient entry to be visible and click
+      await page.getByRole("link", { name: "Patient Home" }).first().click();
+
+      // Wait for patient details to load
+      await expect(
+        page.getByRole("button", { name: "Create Encounter" }),
+      ).toBeVisible();
+
+      await page.getByRole("button", { name: "Create Encounter" }).click();
+      await page.getByRole("button", { name: "Home Health" }).click();
+      await page.getByRole("button", { name: "Create Encounter" }).click();
+      //wait for success message
+      await expect(
+        page.getByText("Encounter created successfully"),
+      ).toBeVisible();
+    });
+
+    //mark encounter as complete
```

Comment:
- verify that the encounter page is showing the home health badge in the encounter switcher and the encounter details card